.github/ISSUE_TEMPLATE/helpdesk-issue.yml

```diff
@@ -0,0 +1,71 @@
+name: Heldesk Issue
+description: Used as output from helpdesk issue triage.
+title: "[HD]: "
+labels: ["helpdesk"]
+assignees:
+  - jadudm
+  - carley-sullivan
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Remember to keep all PII out of tickets.**
+        
+        Do not use names, and do not attach files to this ticket. 
+  - type: input
+    id: zendesk-link
+    attributes:
+      label: Zendesk link
+      description: Link to the issue in Zendesk
+      placeholder: ex. https://fac-something.zendesk.com/something/...
+    validations:
+      required: true
+  - type: checkboxes
+    id: fac-components
+    attributes:
+      label: FAC components involved
+      description: Select all that apply
+      options:
+        - label: submission system (app.fac.gov)
+        - label: workbooks
+        - label: static site (fac.gov)
+        - label: The helpdesk (zendesk)
+        - label: API (api.fac.gov)
+        - label: other
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: What browser did the user report as using?
+      multiple: false
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Other (Opera, Brave, etc.)  
+  - type: markdown
+    attributes:
+      value: |
+        Audits are not yet public at this point. 
+        
+        Places all files and screenshots in the Google Drive Helpdesk folder and link to that folder here.
```

Comment:
This seems like it should have a link directly to the Google Drive Helpdesk folder